### PR TITLE
Fix change character lag, bg resizing and repos weirdness, inconsistent behavior, sticker setting etc.

### PR DESCRIPTION
**Pull request by Crystalwarrior**
See commits for changelog

### CODE REVIEW

**Review by in1tiate — APPROVED** [review]
seems fine to me

**Review by oldmud0 — CHANGES_REQUESTED** [review]
So far I've encountered a crash condition if you set the position with /pos, switch characters to one whose default position is different, and chat.

File: src/courtroom.cpp [hunk 1]
```diff
@@ -3786,12 +3786,12 @@ void Courtroom::handle_wtce(QString p_wtce, int variant)
...
       if (variant == 0) {
         sfx_name = ao_app->get_court_sfx("not_guilty", bg_misc);
-        filename = "notguilty";
+        filename = "notguilty_bubble";
         ui_vp_testimony->kill();
       }
       else if (variant == 1) {
         sfx_name = ao_app->get_court_sfx("guilty", bg_misc);
-        filename = "guilty";
+        filename = "guilty_bubble";
```
[the remark this answers is not in the document]
Comment (Crystalwarrior) [reply src/courtroom.cpp:3780]: yeah there is, it's called animated themes

File: src/lobby.cpp [hunk 2]
```diff
@@ -28,8 +28,8 @@ Lobby::Lobby(AOApplication *p_ao_app) : QMainWindow()
 
   ui_server_list = new QTreeWidget(this);
   ui_server_list->setHeaderLabels({"#", "Name"}); //, "Players"});
-  ui_server_list->hideColumn(0);
-  ui_server_list->setHeaderHidden(true);
+//  ui_server_list->hideColumn(0);
+//  ui_server_list->setHeaderHidden(true);
```
[the remark this answers is not in the document]
Comment (Crystalwarrior) [reply src/lobby.cpp:37]: we talked about this on discord so resolving
(being able to easily reset to show the intended master-server ordering is important, as pressing the "refresh" button to reset that ordering is jank as hell)

**Comment by Crystalwarrior** [comment]
> 
> 
> So far I've encountered a crash condition if you set the position with /pos, switch characters to one whose default position is different, and chat. Haven't finished reviewing everything though.

yeah wasn't this an issue w/ the outdated server software that particular server was using instead? You mentioned they run their soft on this commit https://github.com/AttorneyOnline/tsuserver3/commit/41bd23e1c28c1fb444385ece02ec3464d6a1b054

**Comment by oldmud0** [comment]
Yeah the review is old sorry.

**Review by oldmud0 — APPROVED** [review]
This all looks fine after testing. One thing I haven't really tested are stickers but the feature is trivial and doesn't need much testing anyway.

**Comment by oldmud0** [comment]
Let's get this bad boy merged. Please try to avoid sticking multiple fixes and features in one PR next time, I know you like to go on a roll but this slows things down in the long run.

**Pull request merged**